### ISSUE TITLE
Complete Mechanics Of Warden Political Focus Tree

### ISSUE THREAD

**Issue by RegaSaurus24** [SafeHouse-Studios/Foxhole-Conquest#34]
**IN ORDER TO COMPLETE THIS TASK:** Design document of the Warden political focus tree must first be completed.
[Warden Focus Tree Design Document](https://docs.google.com/document/d/1nBrQorOAaVDKdAS-1GCK_uGADRjIHWgAxXIBYLZnBMk/edit?usp=sharing)

**IN ORDER TO COMPLETE THIS TASK:** Issue #32 & #86 must be completed.

**Task Details:** This task involves inputting all of the mechanics of the focuses according to the documentation. Specifically the political section is the center section of the focus tree. Your job is to go through the focus document for the Warden Nation and ensure that all of the focuses within this central section all have the correct mechanics as is listed in the "Full Warden Focus Tree Designs". When looking for the correct command words used to create specific mechanics first check the two supporting links "HoI4 Modifiers" & "HoI4 Effects". If still unsure about any of the command words or how to use them consult the programming leader.

**Documentation:** [Full Warden Focus Tree Designs](https://docs.google.com/document/d/1SKl4GcNm26qxHmSv5CZwRSjYSlV2vQap/edit)
[Layout Designs Of Full Warden Focus Tree](https://drive.google.com/file/d/13g094-fFBCz2B4QAgWp8MCZZXAoD28tB/view?usp=sharing)
[HoI4 Modifiers](https://mod-coop-hoi4.fandom.com/wiki/Modifiers)
[HoI4 Effects](https://hoi4-modding.fandom.com/wiki/Effects)
[Map Countires](https://drive.google.com/drive/folders/1mrD3alHstaHa7VH5iz8K4TKaR9VRC5Ev?usp=sharing) - waiting on completion of issue #86.
[Map Regions](https://drive.google.com/drive/folders/1V377_6HHvSanDyaJ1nmEXnGV2ix1Y7nZ?usp=sharing) - waiting on completion of issue #86.

**Tutorial:** [Basic Focus Tree Creation](https://www.youtube.com/watch?v=svFVLXzU2MQ&ab_channel=TheIronWorkshop)

**Comment by Mister-Mint** [comment]
Slay the eagle: effects "division recovery rate in core territory" and "division speed in core territory can't be added as stated, since those boosts can't be added exclusive in core

**Comment by LegendGman** [comment]
Events need to be created in order to finish

**Comment by RegaSaurus24** [comment]
Gman can you confirm that Mint's previous comment has been corrected?
If it has and events are the only thing left to be done then this issue can be closed as the event attachment is covered in another issue.

**Comment by LegendGman** [comment]
Mint's comment has been corrected.

**Comment by RegaSaurus24** [comment]
Due to the map changes this task will need to be gone over again with the new states assigned where relevant.
